extensions/objects/templates/list-of-sets-of-html-strings-editor.directive.ts

```diff
@@ -124,11 +121,6 @@ angular.module('oppia').directive('listOfSetsOfHtmlStringsEditor', [
               }
             }
           }
-
-          if (ctrl.selectedRank !== '') {
-            ctrl.maxPrevIndex = math.max(parseInt(ctrl.selectedRank),
```

Comment:
Isn't this needed?

Reply:
This condition would never get triggered due to line 117. All values are initialized with `''`.

Reply:
Ah, ok, but still do you know why `maxPrevIndex` was used here?

Reply:
Hm, even without the changes of this PR, I don't think this condition would ever have triggered since the first line of oninit, changes the value of selectedRank to `''` if it's undefined. 
And since that is the first thing to run when the directive gets instantiated, we can safely assume that selectedRank would always be undefined. 
Does that answer your question, or have I misunderstood?

Reply:
Hmm, yeah, that's true. Ok then, as long as it works with this config.